src/hooks/useToggle.ts

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export function useToggle(initialState = false) {
+  const [value, setIsOn] = React.useState(initialState);
```

Comment:
We typically have the setter be in the format of `set<state value>`.  This would be`setValue` following that convention.

Reply:
Ah I missed that when I was changing `isOn`